xds-api/src/main/proto/com/github/xds/centraldogma/v1/centraldogma_config_source.proto

```
@@ -0,0 +1,31 @@
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+syntax = "proto3";
+
+package com.github.xds.centraldogma.v1;
```

Buf (1.71.0)

[error] Files with package "com.github.xds.centraldogma.v1" must be within a directory "com/github/xds/centraldogma/v1" relative to root but were in directory "client/java-armeria-xds/src/main/proto/com/github/xds/centraldogma/v1".

(PACKAGE_DIRECTORY_MATCH)